### PR TITLE
NuGet survey banner

### PR DESCRIPTION
**Pull request by lyndaidaii**
UI:

bannerfinal

Addresses https://github.com/NuGet/NuGetGallery/issues/8225

### CODE REVIEW

File: src/NuGetGallery/Views/Shared/Gallery/Header.cshtml [hunk 1]
```diff
@@ -8,6 +8,16 @@
...
 
+<div class="container-fluid banner banner-info text-center">
+    <div class="row">
+        <div class="col-sm-12">
+            <span>
+                &#128075; What do you think about NuGet.org? We're looking for feedback from developers like you.&emsp;<a href="https://aka.ms/NuGetSurvey" target="_blank">Take the survey.</a>
+            </span>
+        </div>
+    </div>
+</div>
```
Comment (loic-sharma) [inline src/NuGetGallery/Views/Shared/Gallery/Header.cshtml:19]: I'd consider moving this to be the last banner, after the cookie and warning banners. I'm guessing the cookie banner needs to be the first thing on the page, is that correct @zhhyu? 

@zhhyu does the cookie banner need to be the first thing on the page?

**Review by loic-sharma — APPROVED** [review]
Please double-check with @zhhyu about banner ordering before merging

**Review by joelverhagen — APPROVED** [review]
Answer comment then :shipit:

**Comment by lyndaidaii** [comment]
> Please double-check with @zhhyu about banner ordering before merging

Cookie banner should be on the top. I will move the survey banner after cookie and warning

**Pull request merged**